afdko/Tools/SharedData/FDKScripts/FDKUtils.py

```diff
@@ -9,14 +9,13 @@
 import traceback
 
```

Comment:
line 30: change `fdkScriptsDir = None` to `fdkSharedDataDir = None`

lines 167 - 176: if this is still needed, then change `import FDK` to `import afdko`